packages/core/strapi/lib/services/cron.js

```diff
@@ -1,13 +1,18 @@
 'use strict';
 
 const { Job } = require('node-schedule');
-const { isFunction } = require('lodash/fp');
+const { isFunction, forEach } = require('lodash/fp');
 
 const createCronService = () => {
-  let jobsSpecs = [];
+  let jobsSpecsMap = {};
```

Comment:
Base on my previous comment I don't think we would need all those changes we could keep this simpler